_data/version2-english/glossary.yml

```diff
@@ -8,7 +8,17 @@ All Theoretical REP: |-
     <li>If a fork has occurred and the <a href="#Forking_Period">forking period</a> has ended, then the theoretic REP supply in the universe is just the amount of REP in that universe (since neither REP in the <a href="#Parent_Universe">parent universe</a> nor REP in previous versions can be migrated after a fork completes).</li>
   </ol>
   This value is used to calculate the <a href="#Forking_Threshold">forking threshold</a>.
-Child Universe: <i>under construction</i>
+Child Universe: |-
+  A child universe is a  <a href="#Universe">universe</a> spawned by its <a href="#Parent_Universe">parent universe</a> when a market in the parent universe <a href="#Fork">forks</a>.
+  <br>
+  <br>
+  In a child universe, the <a href="#Forking_Market">forking market</a> is <a href="#Finalized_Market">finalized</a> as the <a href="#Outcome">outcome</a> that aligns with that particular child universe. For example, a <a href="#Market">market</a> has 3 possible outcomes: <code>Yes</code>, <code>No</code>, and <code>Invalid</code>. When the market forks, 3 new universes are created: <code>universe Yes</code>, <code>universe No</code>, and <code>universe Invalid</code>. In each universe, its corresponding outcome becomes the <a href="#Final_Outcome">final outcome</a>, such as in the <code>universe Yes</code> the final outcome of the forking market is <code>Yes</code>, in the <code>universe No</code> the final outcome of the forking market is <code>No</code>.
```

Comment:
Consider putting each sentence on a separate line.  When it is rendered, it should all render as a paragraph if you don't include `<br>` or `<p>` or multiple newlines, but it makes the PR much easier to review.

Reply:
From now on I will do that 🌝